shared/src/metabase/shared/parameters/parameters.cljc

```diff
@@ -115,9 +115,13 @@
   "Given a seq of parameter values, returns them as a single comma-separated string. Does not do additional formatting
   on the values."
   [values]
-  (if (= (count values) 1)
-    (str (first values))
-    (trs "{0} and {1}" (str/join ", " (butlast values)) (last values))))
+  (condp = (count values)
+    1 (str (first values))
+    2 (trs "{0} and {1}" (first values) (second values))
+    (trs "{0}, {1}, and {2}"
+         (str/join ", " (drop-last 2 values))
+         (nth values (- (count values) 2))
+         (last values))))
```

Comment:
not a big deal, but I think

```
(trs "{0}, and {1}" (str/join ", " (butlast values)) (last values)))
```

would've been fine for this case? (agreed that you need to separate out 2 and >2).

Reply:
That's what I originally had, but I changed to a three-element list to make it more obvious for translators. Without context that `{0}` contains multiple elements, `{0}, and {1}` isn't a natural pattern in English so I was concerned it could be confusing for translators who are seeing it in isolation.

Reply:
Awesome explanation, thank you!